api/envoy/config/common/mutation_rules/v3/mutation_rules.proto

```diff
@@ -87,6 +88,12 @@ message HeaderMutationRules {
   google.protobuf.BoolValue disallow_is_error = 7;
 }
 
+// The HeaderForwardingRules structure specifies what headers are
+// allowed to be forwarded to the ext_proc service.
```

Comment:
```suggestion
// allowed to be forwarded to the external callout service.
```
nit: I think, from a forward thinking perspective, this `HeaderForwardingRules` can be used in external callout in general.

Reply:
changed it into external processing server.